docs/model_zoo/segmentation.rst

```diff
@@ -31,7 +31,7 @@ Table of pre-trained models for semantic segmentation and their performance.
   The test script :download:`Download test.py<../../scripts/segmentation/test.py>` can be used for
   evaluating the models (VOC results are evaluated using the official server). For example ``fcn_resnet50_ade``::
 
-    python test.py --dataset ade20k --model-zoo fcn_resnet50_ade --eval
+    python test.py --dataset ade20k --pretrained --model fcn --backbone resnet50 --eval
```

Comment:
why make this change?

Reply:
Hello,
According in test.py given in 
[semantic-segmentation](https://gluon-cv.mxnet.io/model_zoo/segmentation.html#semantic-segmentation)
The args defined in parse_args() no longer has args 'model-zoo'

Reply:
Please add that back instead. Not sure who removed that.

Reply:
I'm not sure whether this may have conflicts with these settings.

https://github.com/dmlc/gluon-cv/blob/a78965fe0f4d6003a1bbd7a9553f6d71205daa5c/scripts/segmentation/test.py#L204-L230